app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -143,6 +143,21 @@ def self.replace_context_secrets(var_name)
         "$$GITHUB_SECRET_#{var_name.upcase}"
       end
 
+      def self.replace_context_runner(var_name)
+        case var_name
+        when "name"
+          "$$BUILDKITE_AGENT_NAME"
+        when "os"
+          "$$BUILDKITE_AGENT_META_DATA_OS"
+        when "arch"
+          "$$BUILDKITE_AGENT_META_DATA_ARCHITECTURE"
+        when /(temp|tool_cache)/
```

Comment:
according to the documentation, `tool_cache` is like `/usr/local/bin` or something like that. Not sure if mapping it to a tempdir is a good idea when there is no direct equivalent as agent environments and available executables are not uniform